client/src/components/EditProfile/ProfilePhoto/useStyles.ts

```diff
@@ -0,0 +1,33 @@
+import { createStyles, makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      '& > *': {
+        margin: theme.spacing(1),
+      },
```

Comment:
I'm assuming this is mainly left over from the material ui examples. I'd recommend removing this as it doesn't add much to the styling of the component as it currently stands (just adds a margin to the `form` element). 

It is also possble to achieve a similar result by using an appropraite flexbox option to spread out each section of the form/ui (e.g. check out [these](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#justify-content)), although keep in mind that you would need to specify `flexDirection: "column"` 👍🏼